cmd/fetch.go

```diff
@@ -68,7 +67,7 @@ func (o *FetchOptions) Run() error {
 
 		items := make([]fmt.Stringer, len(res))
 		for i, r := range res {
-			items[i] = versionInfoStringer(r)
+			items[i] = versionInfoStringer(r.VersionInfo)
```

Comment:
I wanted to start a small discussion here @b5 @dustmop. Per the intention of the PR this should also be converted to a `DatasetLogItem` and use the `dsLogItemStringer` however that changes the output of what we tend to show with the versionInfo. Now we have a choice of whether to appropriate the `dsLogItemStringer` to show a more comprehensive output and similar to `versionInfoStringer`, agree to keep the current setup (which is not in line of what we wanted from fetch) or to accept the new output provided by the current implementation of `dsLogItemStringer`.

In either case, with this the difference between `qri log` and `qri fetch` is becoming non-existant as they both come by the same process from the logbook, just two different branches and now also share the same output.

Reply:
Also wondering whether we should also follow suit for `qri list` as VersionInfo is intended more as an internal structure and convert the output format to `DatasetLogItem` as well. Also `qri list` no longer sports `CommitTitle` and `CommitMessage` either.

Reply:
@ramfox Just wanted to get your eye in this and see if it alignes with expectations?
So `qri fetch/log` will return the `DatasetLogItem` which contains the `CommitTitle` and `CommitMessage`. 
`qri list` (while unsure if it had it before) will be losing `CommitTitle` and `CommitMessage`.

Reply:
Got it! This works with desktop expectations. Thanks :)